### DISCUSSION TITLE
Grouping of Rows

### DISCUSSION THREAD

**Discussion by jigapate** [Power-Components/livewire-powergrid#1449] [General]
Hello,

Is there possibility of grouping row based on column values?
Say there are many X in column, and need to group all rows with X?

Thanks,

**Comment by dansysanalyst** [comment]
Hi @jigapate,

I was thinking about having this feature some weeks ago on I project I was working. 

Could you please show me an example of how could it be useful in your project and what your data would look like?

Just to pick our brains...

Thanks

**Reply by jigapate** [reply]
I am using Powergrid to show stock data where I have a column with 4 possible values,
If we have a grouping of with those column values, I can easily see what rows are in those specific values,

You can see in the below screenshot, that column oi sense.
If I am interested in a specific value only then I can easily see it, instead of searching, and even if we get the total count of it, it would be double beneficial. 

In my earlier project I was using datatables where this feature was there, https://datatables.net/examples/advanced_init/row_grouping.html

![image](https://github.com/Power-Components/livewire-powergrid/assets/11524768/08e0cdb9-1810-40b1-bf85-5af92882918f)
